tests/unit/h/services/group_create_test.py

```diff
@@ -170,10 +185,25 @@ def test_it_sets_access_flags(self, svc, creator, origins, flag, expected_value)
 
         assert getattr(group, flag) == expected_value
 
+    @pytest.mark.parametrize("pass_kwarg", [True, False])
     def test_it_creates_group_with_no_organization_by_default(
-        self, creator, svc, origins
+        self, creator, svc, origins, pass_kwarg
     ):
-        group = svc.create_open_group("Anteater fans", creator.userid, scopes=origins)
+        """
+        If the caller passes no organization it creates a group with no organization.
+
+        This works both if the caller omits the `organization` kwarg entirely
+        and if the caller passes `organization=None`. It's convenient for the
+        caller to be able to do it either way.
+        """
+        if pass_kwarg:
+            kwargs = {"organization": None}
+        else:
+            kwargs = {}
+
+        group = svc.create_open_group(
+            "Anteater fans", creator.userid, scopes=origins, **kwargs
+        )
```

Comment:
Repetitive tests, but that's how the existing tests are structured and it's a result of how the code is structured. I prefer the test repetition over getting clever in the tests, and don't want to take the time to restructure the code and rewrite all the tests right now.